Cargo.lock

```diff
@@ -0,0 +1,1156 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Does this *really* need to be checked in? It seems not to me.

Reply:
Actually no way should this be checked in.

Reply:
I think it might be interesting for reproducible builds, see also [this](https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html)